sources/host-ctr/go.mod

```diff
@@ -3,27 +3,32 @@ module host-ctr
 go 1.12
 
 require (
-	github.com/aws/aws-sdk-go v1.34.3 // indirect
-	github.com/awslabs/amazon-ecr-containerd-resolver v0.0.0-20200804232255-8c110f606e9e
-	github.com/containerd/cgroups v0.0.0-20200710171044-318312a37340 // indirect
+	github.com/Microsoft/go-winio v0.4.15 // indirect
+	github.com/Microsoft/hcsshim v0.8.10 // indirect
```

Comment:
If you run `go mod tidy`, does it trim these dependencies?

Reply:
No, it doesn't

Reply:
`go mod why`s
```
$ go mod why github.com/Microsoft/go-winio
# github.com/Microsoft/go-winio
host-ctr/cmd/host-ctr
github.com/containerd/containerd/cio
github.com/Microsoft/go-winio
$ go mod why github.com/Microsoft/hcsshim
# github.com/Microsoft/hcsshim
host-ctr/cmd/host-ctr
github.com/containerd/containerd
github.com/containerd/containerd/archive
github.com/Microsoft/hcsshim

```

Reply:
That's what I suspected - I'm not leery of these dependencies lingering in this case.

Reply:
I was hoping they'd be trimmed since they're Windows-specific, but oh well...